ui/StatusQ/include/StatusQ/movablemodel.h

```diff
@@ -22,22 +22,25 @@ class MovableModel : public QAbstractListModel
     QVariant data(const QModelIndex &index, int role) const override;
     QHash<int, QByteArray> roleNames() const override;
 
-    Q_INVOKABLE void detach();
+    Q_INVOKABLE void desyncOrder();
```

Comment:
suggestion: use slots for setters and leave `Q_INVOKABLE` only for getters

Reply:
Is there any difference between them? In both cases invokable methods like slots are made available through the metatype system. The only difference I can see is semantic, those methods are rather intended to be called from scripts instead of connected via signals/slots. For that reason Q_INVOKABLE works ok for me here.

Reply:
There's no difference really, if you want to expose it to QML, either make it a slot or mark it with `Q_INVOKABLE`